bootstrap/005_table_migrations.sql

```diff
@@ -111,7 +120,40 @@ begin
     let stmt varchar;
     call internal.generate_insert_statement_cmd(:target_schema, :target_table, :source_schema, :source_table, :where_clause) into stmt;
     execute immediate stmt;
+    SYSTEM$LOG_INFO('Running INSERT with generated query: ' || :stmt);
     let inserted number := (select * from TABLE(RESULT_SCAN(LAST_QUERY_ID())));
   return :inserted;
 end;
 $$;
+
+
+create or replace procedure internal.generate_ctas_statement(target_schema varchar, target_table varchar, source_schema varchar, source_table varchar, where_clause varchar)
```

Comment:
we don't need ctas anymore do we? We agreed to re-order? Or am I missing something

Reply:
Yeah, i just didn't unwind enough commits. CTAS shouldn't need this indirection anymore.